src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -175,6 +175,9 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
     });
 
     webviewPanel.onDidChangeViewState(e => {
+      if (webviewPanel.visible) {
```

Comment:
When focused document is changed in CfgEditorPanel, it's shown in OneExplorer's tree view.